ImageLoader.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name         = "ImageLoader"
-  s.version      = "0.9.0"
+  s.version      = "0.10.0"
```

Comment:
Might be good to add something in the readme to use 0.9.0 for pre swift 3 use and 0.10.0 for post.